code/datums/diseases/advance/symptoms/vision.dm

```diff
@@ -75,3 +75,97 @@ Bonus
 						eyes.forceMove(get_turf(M))
 				else
 					to_chat(M, "<span class='userdanger'>Your eyes burn horrifically!</span>")
+
+/datum/symptom/ocularsensitivity
+	name = "Ocular Hyper-reception"
+	desc = "The virus increases the sensitivity of the rods and cones to light, allowing better vision in the dark"
+	stealth = 1
+	resistance = -2
+	stage_speed = -2
+	transmittable = 0
+	level = 9
+	severity = 0
+	base_message_chance = 50
+	symptom_delay_min = 15
+	symptom_delay_max = 50
+	var/better_vis = FALSE
+	var/nvgs = FALSE
+	var/thermal = FALSE
+	var/xray = FALSE
+	var/nvision_buff = FALSE //Toggles TRUE when the Night Vision Buffs are Applied
+	var/other_buff = FALSE //Toggles TRUE when Thermal/Xray is applied
```

Comment:
All those vars could be just one list

Reply:
Cramming them into an associative list so it's easier to understand when reading.